e2e-tests/pitr/run

```diff
@@ -6,6 +6,15 @@ set -o xtrace
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 
+if [[ "${IMAGE_PXC}" == *"percona-xtradb-cluster-operator"* ]]; then
+    PXC_VER=$(echo -n "${IMAGE_PXC}" | $sed -r 's/.*([0-9].[0-9])$/\1/')
+else
+    PXC_VER=$(echo -n "${IMAGE_PXC}" | $sed -r 's/.*:([0-9]+\.[0-9]+).*$/\1/')
+fi
+if [[ "${PXC_VER}" == "5.7" ]]; then
```

Comment:
we can keep it simple
```suggestion
if [[ "$IMAGE_PXC" =~ 5\.7 ]]; then
```

Reply:
Agree. I wanted to escape the issue with some images like `percona/percona-xtradb-cluster:5.7.32-31.47` where the last two numbers change often, but not sure how likely is it to happen to have a match.